user/distrobox/patches/tac.patch

```diff
@@ -0,0 +1,13 @@
+diff --git a/distrobox-enter b/distrobox-enter
+index 471142b..be4d774 100755
+--- a/distrobox-enter
++++ b/distrobox-enter
+@@ -709,7 +709,7 @@ if [ "${unshare_groups:-0}" -eq 1 ]; then
+ fi
+ 
+ # Generate the exec command and run it
+-cmd="$(generate_enter_command | tac)"
++cmd="$(generate_enter_command | awk '{a[i++]=$0} END {for (j=i-1; j>=0;) print a[j--] }')"
```

Comment:
unless you're upstreaming this, `tail -r` is shorter and clearer

Reply:
it is in https://github.com/89luca89/distrobox/pull/1591 , so fine for now